src/include/distributed/commands.h

```diff
@@ -230,10 +224,12 @@ extern List * PreprocessAlterDatabaseRefreshCollStmt(Node *node, const char *que
 													 ProcessUtilityContext
 													 processUtilityContext);
 
-
 extern List * PreprocessAlterDatabaseSetStmt(Node *node, const char *queryString,
 											 ProcessUtilityContext processUtilityContext);
 
+extern List * PostprocessCreateDatabaseStmt(Node *node, const char *queryString);
+extern List * PreprocessDropDatabaseStmt(Node *node, const char *queryString,
+										 ProcessUtilityContext processUtilityContext);
```

Comment:
Can we avoid unnecessary style changes, i.e., means revertig the changes other those two lines in this file?